Source/UI/IdentityUIRoute.swift

```diff
@@ -29,6 +29,62 @@ extension IdentityUI {
 }
 
 extension IdentityUI.Route {
+
+    enum TypeID: String {
```

Comment:
Is this a 1:1 mirror of `Identity.Route`? Could it be a computed property instead?